--- configure	2014-08-26 22:11:17.000000000 +0100
+++ configure.new	2014-12-13 09:57:33.526960551 +0000
@@ -9795,10 +9795,8 @@
 
     break
 done
-if test x"$ac_cv_lib_avformat_av_open_input_file" != x"yes" &&
-   test x"$ac_cv_lib_avformat_avformat_open_input" != x"yes"; then
-   as_fn_error $? "Could not find libavformat - part of ffmpeg" "$LINENO" 5
-fi
+# XXX hack it in
+LIBAVFORMAT_LIBS="-lavformat -lavcodec"
 
 
 as_ac_Lib=`$as_echo "ac_cv_lib_avutil ''_av_rescale_q" | $as_tr_sh`
@@ -9842,7 +9840,8 @@
 if eval test \"x\$"$as_ac_Lib"\" = x"yes"; then :
   LIBAVUTIL_LIBS="-lavutil"
 else
-  as_fn_error $? "Could not find libavutil - part of ffmpeg" "$LINENO" 5
+  # XXX hack it in
+  LIBAVUTIL_LIBS="-lavutil -lz -lm -lswresample -lopus -lrt"
 fi
 
 
